xlsynth-g8r/src/equiv/boolector_backend.rs

```diff
@@ -0,0 +1,550 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#![cfg(feature = "with-boolector-built")]
+
+use std::{
+    collections::HashMap,
+    ffi::{CStr, CString},
+    io,
+    sync::Arc,
+};
+
+use boolector_sys::*;
```

Comment:
I think `use namespace::*` is not best practice in rust, better to fully qualify names or list out which ones are used.